proto/ibc/core/channel/v1/tx.proto

```diff
@@ -208,6 +212,27 @@ message MsgChannelCloseConfirm {
 // type.
 message MsgChannelCloseConfirmResponse {}
 
+// MsgSendPacket sends an outgoing IBC packet
+message MsgSendPacket {
+  option (cosmos.msg.v1.signer) = "signer";
+
+  option (gogoproto.goproto_getters) = false;
+
+  string                    port_id           = 1;
+  string                    channel_id        = 2;
+  ibc.core.client.v1.Height timeout_height    = 3 [(gogoproto.nullable) = false];
+  google.protobuf.Timestamp timeout_timestamp = 4 [(gogoproto.nullable) = false, (gogoproto.stdtime) = true];
```

Comment:
might be better to use uint64 here as thats what's used in `Packet` proto def. 

we do use `google.protobuf.Timestamp` in tm consensus states tho, there is some inconsistency there, but better to be consistent with packet I think